README.md

```diff
@@ -439,6 +439,35 @@ Make sure you've completed the following assignments:
 * (Optional) Go to your user dashboard and create a new repository to practice what you've learned here. Feel free to experiment!
 * [Let us know what worked and what needs improvement in this lesson in the CoMSES Education forum](https://forum.comses.net/c/education/28). Was there anything confusing? What can we do better? If you'd like to submit feedback in private, please send an email to us at `education@comses.net`
 
+## 💪 Contributing to this repo
+
+You can train your GitHub skills by contributing to this repo.
+
+Fork this repository on GitHub. You do that by clicking on `Fork`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219558561-441e6462-035c-4807-932b-bff465e98968.png)<br/><br/>
+Name your fork something informative by changing the text under `Repository name` and then click on `Create fork`:
+![image](https://user-images.githubusercontent.com/84760072/219558663-8f2a6a63-fa10-428b-99a0-ffb316b3a521.png)<br/><br/>
+After forking the repository, you will be in your forked repository. Which looks something like this:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219558813-56130098-7f28-4b18-9d60-5f29d01fd1a0.png)<br/><br/>
+Make sure your forked repository is in sync with the upstream branch. You will have to do this periodically to ensure that your fork is up to date. You do that by clickin on `Sync fork`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219559314-5b2da1ee-2020-44f5-9555-8d0d345e3f36.png)<br/><br/>
+If your fork is out of date, this box will appear. Compare the changes by clicking on `Compare`. Once you are satisfied with the changes, click on `Update Branch`:<br/>
+![image](https://user-images.githubusercontent.com/84760072/219559801-46ab912b-5910-4e61-b5e1-9885b963ace1.png)<br/><br/>
+You don't want to make commits straight to your main branch so make a new branch. You do that by clicking on `[number] branch`. [number] can be any number, all that matters is you click on it.<br/>
```

Comment:
Should explain why you don't want to make commits straight to your main branch

ELI5